src/modules/abtest/finish.ts

```diff
@@ -21,6 +21,7 @@ const promptWorkspaceToFinishABTest = () =>
   abtester
     .status()
     .then(map(({ WorkspaceB }) => WorkspaceB))
+    .then(filter((workspace: any) => workspace !== undefined))
     // @ts-ignore
```

Comment:
Please, remove ts-ignore comment.

Reply:
I'll wait for the resolution of [the other conversation](https://github.com/vtex/cli-plugin-abtest/pull/9#discussion_r611680482). If we replace `any` with `string | undefined` as suggested there, it seems that we would need this `// @ts-ignore` - even if we add another `// @ ts-ignore` right after line `23`.

Reply:
Ok! Please see my comment, I've posted a workaround there.